CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Spread (jitter) re-queueing of reports by +/- 10% by default to help smooth resource utilization.
+- Added missing monitoring options in the Helm chart values.yaml
```

Comment:
Can you please move this under a new `### Added` subheading under `[Unreleased]`? The release headings are automatically generated, so 0.4.1 refers to a release which is already out.

Reply:
My bad, should be good now :+1: